components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/AuthorizedAPIManagementServiceImpl.java

```diff
@@ -169,23 +170,7 @@ public void patchAuthorizedAPI(String appId, String apiId, List<String> addedSco
                                    List<String> removedScopes, String tenantDomain)
             throws IdentityApplicationManagementException {
 
-        ApplicationAuthorizedAPIManagementEventPublisherProxy publisherProxy =
-                ApplicationAuthorizedAPIManagementEventPublisherProxy.getInstance();
-        publisherProxy.publishPreUpdateAuthorizedAPIForApplication(appId, apiId, addedScopes, removedScopes,
-                tenantDomain);
-        Collection<AuthorizedAPIManagementListener> listeners = ApplicationMgtListenerServiceComponent
-                .getAuthorizedAPIManagementListeners();
-        for (AuthorizedAPIManagementListener listener : listeners) {
-            listener.prePatchAuthorizedAPI(appId, apiId, addedScopes, removedScopes, tenantDomain);
-        }
-        authorizedAPIDAO.patchAuthorizedAPI(appId, apiId, addedScopes, removedScopes,
-                IdentityTenantUtil.getTenantId(tenantDomain));
-        updateRolesWithRemovedScopes(appId, removedScopes, tenantDomain);
-        for (AuthorizedAPIManagementListener listener : listeners) {
-            listener.postPatchAuthorizedAPI(appId, apiId, addedScopes, removedScopes, tenantDomain);
-        }
-        publisherProxy.publishPostUpdateAuthorizedAPIForApplication(appId, apiId, addedScopes, removedScopes,
-                tenantDomain);
+        this.patchAuthorizedAPI(appId, apiId, addedScopes, removedScopes, null, null, tenantDomain);
```

Comment:
cant we pass empty arrays instead of null

Reply:
Fixed in 2d72c264576eb9e95bb40ef8fef70f8cdd5cbeb1